luigi_pipeline/seqr_mito_loading.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import sys
+import pkg_resources
+import pprint
+
+import luigi
+import hail as hl
+
+from lib.hail_tasks import HailElasticSearchTask
+from lib.model.mito_mt_schema import SeqrMitoVariantsAndGenotypesSchema
+from sv_pipeline.utils.common import get_es_index_name
+import seqr_loading
+
+logger = logging.getLogger(__name__)
+
+
+class SeqrMitoMTTask(seqr_loading.SeqrVCFToMTTask):
```

Comment:
it would be cleaner if you imported `SeqrVCFToMTTask` and used it here, instead of referencing it on the module